HW1/P5/P5.txt

```diff
@@ -1 +1,14 @@
 # Your discussion here
+
+1. Short Path
+1) ”Kevin_Bacon" -> "Harvard University"
+Using Labels : 2729536 -> 2578703 -> 2152782
+Using Paget Titles : Kevin_Bacon -> John_Lithgow -> Harvard_University
+
+2) "Harvard University" -> "Kevin Bacon"
+Using Labels : 2152782 -> 4530382 -> 44868 -> 2729536
+Using Paget Titles : Harvard_University -> September_8 -> 1958 -> Kevin_Bacon
```

Comment:
I got the same number of steps, but I did not get the same path. I think it is OK though to have multiple different answers to this as there may be different paths one can take.